tools/oesign/oesign.c

```diff
@@ -269,10 +262,24 @@ static int _load_pem_file(const char* path, void** data, size_t* size)
     return rc;
 }
 
+static int _load_pem_file(const char* path, void** data, size_t* size)
+{
+    int err = _load_file(path, data, size);
+    if (err == 0)
+    {
+        /* Zero terminate the PEM data. */
+        uint8_t* data_tmp = (uint8_t*)*data;
+        data_tmp[*size] = 0;
```

Comment:
Is this safe? Would *data have space for this null terminator?

Reply:
`_load_pem_file()` takes a dependency on `_load_file()` which always allocates the extra size needed for null-termination.